README.md

```diff
@@ -29,7 +37,7 @@ npm i
 ### Set environment variables
 
 ```bash
-cp .env.example .env
+cp .env.development .env.development.local
```

Comment:
actually should we just rename the env file to `.env.local`? Not really development env vars also.

Reply:
You still need a local copy of the file that is under `.gitignore`. The last thing you want would be for the user to commit and push config (or secrets!) into their git remote..

Reply:
good point